docs/release/trg-0/trg-4-08.md

```diff
@@ -0,0 +1,49 @@
+---
+title: TRG 4.08 - Multi-platform
+---
+
+| Status | Created     | Post-History                              |
+|--------|-------------|-------------------------------------------|
+| Draft  | 14-Mar-2024 | Initial contribution                      |
+
+## Why
+
+Providing container images that support multiple CPU architectures enhances flexibility, compatibility, and efficiency across hardware devices.
+
+This is especially beneficial for supporting use cases like tutorials and local playgrounds.
+
+## Description
+
+Container images **must** be built for **amd64 and arm64** CPU architectures.
+
+### Implementation
+
+Add and enhance the GitHub Workflow with the following steps:
+
+```yaml
+name: Build - Docker image (SemVer)
+
+...
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+
+    steps:
+      ...
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+
+      - name: Build and push
+        uses: docker/build-push-action@v3
+        with:
+           context: .
+           platforms: linux/amd64, linux/arm64
```

Comment:
we already have two examples for build and push worflows 
https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-05
https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-01

maybe we could add the `platforms: linux/amd64, linux/arm64` directly within one of theese two trg´s and adding the mutli-platform info in there, or do you really want to have a separate mutli-platform TRG?

Reply:
I consolidated to one example example workflow in TRG 4.01